core/src/modules/plugin.ts

```diff
@@ -189,7 +191,7 @@ export namespace plugin {
       data: mappedRecordProperties,
       creatorType: "run",
       creatorId: run.id,
-    });
+    } as CreationAttributes<Import>);
```

Comment:
I'm glad this needs types, but shouldn't it not need the `as` if it's passing what's necessary?